### PR TITLE
Update VLC to 3.0.11

### PR DESCRIPTION
**Pull request by vFlagR**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/2020)

### CODE REVIEW

File: manifests/VideoLAN/VLC/3.0.11.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@ Description: A free and open source cross-platform multimedia player and framewo
 Homepage: https://www.videolan.org/vlc/
 Installers:
   - Arch: x64
-    Url: https://download.videolan.org/pub/videolan/vlc/last/win64/vlc-3.0.10-win64.msi
-    Sha256: B15A8B8BBF0027542AFE060CD00C6105D34A311A7E06DB028B62BB500B3DDD8E
+    Url: https://download.videolan.org/pub/videolan/vlc/last/win64/vlc-3.0.11-win64.msi
```
Comment (OfficialEsco) [inline manifests/VideoLAN/VLC/3.0.11.yaml:14]:
```suggestion
    Url: https://download.videolan.org/pub/videolan/vlc/3.0.11/win64/vlc-3.0.11-win64.msi
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Static link update from #2005

**Comment by qiancy98** [comment]
Maybe you have to creat a new file naming 3.0.11.yaml? Instead of changing the 3.0.10.yaml...

**Comment by OfficialEsco** [comment]
No idea why there is a conflict, but seems like there is something wrong with renaming the manifest name, even tho i did it yesterday on #2055

**Review by qiancy98 — CHANGES_REQUESTED** [review]
Use these two files instead. Consider changing files to be deleting one and then creating one: so in two branches you both delete that file: this turn into a conflict.
[files.zip](https://github.com/microsoft/winget-pkgs/files/4825117/files.zip)

I'm a newer to git, and that's my assumption. Maybe you can have a try?

**Comment by qiancy98** [comment]
It seems that the bot thought only the reviewers would make a review.

**Comment by KevinLaMS** [comment]
Duplicate of #2068

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**